### ISSUE TITLE
wandb config for models

### ISSUE THREAD

**Issue by espinielli** [niclaswue/ATOW-Prediction#8]
from `run_wandb.py` it is not clear which models are defined in `wandb_config`...or at least I was not able to find the relevant info.
(yes I see there is s `models/` directory but it is not clear what is used)

**Comment by wurzelDeveloper** [comment]
```run_wandb.py``` by default uses the ```AutogluonModel``` (see lines 63 to 70):

```python
model_config = {
    "time_limit": args.time,
    "preset": args.quality,
    "verbosity": 2,
}

evaluator = MetricEvals()
model = AutogluonModel(**model_config)
loader = DataLoader(Path("data"))
```
Where ```args``` are command line arguments to configure the training (use ```run_wandb.py --help``` for more info).

The model is also explained in the [Project Overview](https://github.com/niclaswue/ATOW-Prediction/blob/main/documentation/project_overview.md#general-overview).
I'll keep this issue open as a reminder for us to improve the documentation, as it seems not clear enough right now.

**Comment by wurzelDeveloper** [comment]
- [ ] Consider updating the readme with more information on how to run the training with command line arguments and with information about what model is used

**Comment by niclaswue** [comment]
Yes, just as an additional comment, we started with the simpler sklearn model class and the average ensemble, however autogluon does all that under the hood and outperformed the sklearn models by a large margin. Therefore we only used Autogluon afterwards. It would also be an option to delete the old model classes.

**Comment by espinielli** [comment]
great thank you.
I created the issue (as per other repos of the PRC Data Challenge) because I think it is good to give others the possibility to understand, learn and reuse.
Great job!
